static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-native-vs-dfp.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+import { makeABTest } from 'common/modules/commercial/contributions-utilities';
+import type { AdBlockEpicTemplate } from 'common/modules/commercial/contributions-utilities';
+import { createSlots } from 'commercial/modules/dfp/create-slots';
+
+// The adblockInUse boolean in detect.js is (necessarily) a promise.
+// This does not interact well with the synchronous canRun() function expected by the AB testing framework.
+// To mitigate against this, create a function which returns false <=>
+// ad blocker is active or it is unresolved whether it is active.
+// Since the test running is predicated on this function returning true,
+// readers in the test will have ad blocker off,
+// but we might not include all readers who have ad blocker off.
```

Comment:
Do we have a sense of who will be excluded? Is it a "random" (i.e. unpredictable) race condition or is it something systematic (e.g. anyone who's on their first visit)?

Reply:
random

Reply:
worth noting, when I have been developing locally, `isAdBlockerOff()` has always returned the correct value